### DISCUSSION TITLE
[FEATURE REQUEST] Add stopped watching

### DISCUSSION THREAD

**Discussion by allanabiud** [trakt/showly#609] [Ideas / Feature Requests]
This app is amazing and does everything I want except for two things
1. Having a friends feature to follow friends and also see what they're watching (though I know this is difficult and will probably never be done.)
2. And secondly, a stopped watching section for tv shows that you stopped watching.

**Comment by michaldrabik** [comment]
Hi.

1. Not thinking of adding any social features at this moment. Showly was meant to be an alternative for social-packed apps like TvTime. Not everyone likes that.

2. You have 2 options. 1st is simply create your own Custom List for that and move these shows there. 2nds is to use already present Hidden section that is syncable with Trakt.
